src/AaveV3Update_20231002weth/DeployAaveV3Update_20231002weth.s.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {GovHelpers} from 'aave-helpers/GovHelpers.sol';
+import {
+  EthereumScript,
+  OptimismScript,
+  ArbitrumScript
+} from 'aave-helpers/ScriptUtils.sol';
+import {
+  AaveV3OptimismUpdate20231002wethPayload,
+  AaveV3ArbitrumUpdate20231002wethPayload
+} from './AaveV3Update_20231002weth.sol';
+
+contract CreateProposal is EthereumScript {
+  function run() external broadcast {
+    GovHelpers.Payload[] memory payloads = new GovHelpers.Payload[](2);
+    payloads[0] = GovHelpers.buildOptimism(0x0f36Dda2b47984434051AeCAa5F9587DEA7f95B7);
+    payloads[1] = GovHelpers.buildArbitrum(0xE1Dd796dBEB5A67CE37CbC52dCD164D0535c901E);
+    GovHelpers.createProposal(payloads, 0x6762e698b21746471fa324dcecc217dc65e245d8bd850b7d31a743c24aec9ff4);
```

Comment:
we recommend using aave-proposals repo & the ipfs helper as there's no good way to ensure that this hash in fact is aligned with the md you posted above.